NuRadioReco/modules/io/coreas/coreas.py

```diff
@@ -196,10 +310,56 @@ def make_sim_station(station_id, corsika, observer, channel_ids, weight=None):
     sim_station.set_simulation_weight(weight)
     return sim_station
 
+def create_sim_station(station_id, evt, weight=None):
+    """
+    creates an NuRadioReco sim station with the information from an event object created with e.g. read_CORSIKA7().
+    An weight per station can be added.
+    To add an electric field the function add_electric_field_to_sim_station() has to be used.
+
+    Parameters
+    ----------
+    station_id : station id
+        the id of the station to create
+    evt : Event object
+        event object containing the CoREAS output, e.g. created with read_CORSIKA7()
+    weight : weight of individual station
+        weight corresponds to area covered by station
+
+    Returns
+    -------
+    sim_station: sim station
+        simulated station object
+    """
+    coreas_sta = evt.get_station(station_id=0) # read_coreas has only station id 0
+    coreas_sim_station = coreas_sta.get_sim_station()
+
+
+    sim_station = NuRadioReco.framework.sim_station.SimStation(station_id)
+    sim_station.set_parameter(stnp.azimuth, coreas_sim_station.get_parameter(stnp.azimuth))
+    sim_station.set_parameter(stnp.zenith, coreas_sim_station.get_parameter(stnp.zenith))
+    sim_station.set_parameter(stnp.cr_energy, coreas_sim_station.get_parameter(stnp.cr_energy))
+    sim_station.set_parameter(stnp.cr_xmax, coreas_sim_station.get_parameter(stnp.cr_xmax))
+    magnetic_field_vector = coreas_sim_station.get_magnetic_field_vector()
+    sim_station.set_magnetic_field_vector(magnetic_field_vector)
+    try:
+        sim_station.set_parameter(stnp.cr_energy_em, coreas_sim_station.get_parameter(stnp.cr_energy_em))
+    except:
+        global warning_printed_coreas_py
+        if(not warning_printed_coreas_py):
+            logger.warning("No high-level quantities in HDF5 file, not setting EM energy, this warning will be only printed once")
+            warning_printed_coreas_py = True
+    if coreas_sim_station.is_cosmic_ray():
+        sim_station.set_is_cosmic_ray()
+
+    sim_station.set_simulation_weight(weight)
+    return sim_station
 
 def make_sim_shower(corsika, observer=None, detector=None, station_id=None):
```

Comment:
Same comment as above, I think we can simply remove this one

Reply:
@cg-laser , @fschlueter ?

Reply:
I commented above.